notifikasjon/src/main/kotlin/no/nav/helsearbeidsgiver/inntektsmelding/notifikasjon/service/ManuellOpprettSakService.kt

```diff
@@ -44,7 +44,6 @@ class ManuellOpprettSakService(
     override val startKeys =
         setOf(
             Key.FORESPOERSEL_ID,
-            Key.UUID,
```

Comment:
Hva slags service er egentlig denne `ManuellOpprettSakService`? Det ser ikke ut som det finnes en producer i Simba som lager meldinger med eventnamet MANUELL_OPPRETT_SAK_REQUESTED. Betyr det at disse meldingene lages et sted utenfor Simba der vi også har fjernet UUID-keyen?

Reply:
`ManuellOpprettSakService` ble laget en gang fordi systemet hadde trynt i under oppretting av en sak, så ønsket vi en måte å opprette saken manuelt på, for å rydde opp. Det er lenge siden den har blitt brukt, og med stateless kan vi trolig fjerne den (siden det var Redis-timeout som var problemet).

Vi produserer meldingene manuelt ved å bruke https://github.com/navikt/hag-cli.

`Key.UUID` trengs ikke defineres som startnøkkel, fordi den allerede er påkrevd i alle servicer. I dette tilfeller ødelegger den også, så derfor fjernes den.

Reply:
Aha, skjønner. Takk for forklaring 😄